app/pandas.ipynb

```diff
@@ -208,18 +220,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "current_year = 2024\n",
+    "df[\"Birth year\"] = current_year - df[\"Age\"]"
```

Comment:
This block of code is repeated from Task 4.1. It seems like a mistake since Task 4.2 requires adding a new calculated field 'Average age', not 'Birth year'. Consider removing or replacing this code with the correct implementation for Task 4.2.